snippets/price.liquid

```diff
@@ -30,13 +30,30 @@
   endif
 -%}
 
+{% assign CustomTags = settings.custom_badges  | newline_to_br | split: '<br />' %}
+{% assign tagLabel = 0 %}
+{% assign hasTag = 0 %}
+
+{% for tag in CustomTags %}
+  {% assign tagSplit = tag | split: ':' %}
+  {% assign pTag = tagSplit.first | strip %}
+  {% assign pLabel = tagSplit.last | strip %}
+
+  {% assign hasTag = product.tags | where: pTag %}
+
+  {% if hasTag.size > 0 %}
+    {% assign tagLabel = pLabel %}
+  {% endif %}
+{% endfor %}
+
```

Comment:
This is not needed. Should be as simple as:
```
{%- liquid
if product.type == 'Wholesale'
  show_badges = true
endif
-%}

then below where `show_badges` is used just add a little bit of code to change the badge to "Partner price" instead of "sale"